deny.toml

```diff
@@ -60,12 +60,15 @@ skip-tree = [
     # wait for criterion to upgrade
     { name = "itertools", version = "=0.10.5" },
 
-    # wait for backtrace and multiple dependents to upgrade
-    { name = "miniz_oxide", version = "=0.6.2" },
-
     # wait for h2 and tower to upgrade
     { name = "indexmap", version = "=1.9.3" },
 
+    # wait for rocksdb to upgrade
+    { name = "bindgen", version = "=0.65.1" },
```

Comment:
Optional: The upgrade to 0.66.1 is allowed by `zcash_script`, but it doesn't require it. So we could stay on bindgen 0.65.1 for both `zcash_script` and `rocksdb`.

Reply:
It's a relatively large crate, good catch!

Reply:
I made the `zcash_script` dependency flexible for exactly this reason.